example/gatsby-config.js

```diff
@@ -12,6 +12,8 @@
 
 module.exports = {
   siteMetadata: {
+    title: "Example Site",
```

Comment:
Why is title and description required ?

Reply:
It helps a lot with search algolia’s search results relevance algorithms. But since you mention it, I think it would be interesting to do search tests with and without titles and descriptions. In the code, I already check to see if there’s a title I can pull from the metadata, and if not, I add the Heading 1 of the page as the title. So I could do the same thing for the description: just add the content snippet as the description for the topic. Or leave out the description all together. I know no one wants to fill out the values in retrospect, so what the hell, let's remove them as requirements in the front matter. Then there's only the removal of keywords left for people to do. I'm totally fine with that as a way to get things out there quicker without killing the product teams. Good call! 😎

Reply:
Hah! You helped me think in another direction without intending to I'm such. To answer your question specifically, in the site Meta data of the config file it was either an experiment I didn't remove, or something I'm still contemplating the need for in the records creation at build time, not runtime. I'll have to look when I get back to my machine.